packages/common/src/internal-non-workflow/codec-helpers.ts

```diff
@@ -80,6 +91,18 @@ export async function encodeToPayload(converter: LoadedDataConverter, value: unk
   return await encodeSingle(payloadCodecs, payloadConverter.toPayload(value));
 }
 
+/**
+ * Run {@link PayloadConverter.toPayload} on an optional value, and then encode it.
+ */
+export function encodeOptionalToPayload(
```

Comment:
nit: This function does not belong in `codec-helpers.ts`, and should not be called `encode`, as it is only _converting_ payloads, not running codecs.

Reply:
renamed and moved to `payload-converter.ts`